src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.skia.cs

```diff
@@ -647,11 +647,11 @@ protected override void OnRightTapped(RightTappedRoutedEventArgs e)
 				_contextMenu.Opened += (_, _) => UpdateDisplaySelection();
 
 				// TODO: port localized resources from WinUI
```

Comment:
We can also drop this TODO 😄

Reply:
Well, let's let auto-merge do its thing. The whole context menu thing will need to be reworked after the port anyway.